upnext/scrape.py

```diff
@@ -0,0 +1,48 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import time
+from bs4 import BeautifulSoup
+from .models import Party, Track
+import spotipy
+import tokens
+
+
+def get_currently_playing(party, embed):
+  token_info = tokens.token_read()
+  sp = spotipy.Spotify(auth=token_info['ACCESS_TOKEN'])
+  # playlist_url = get_playlist_url()
+  party_id = party.uri.split(':')[-1]
+
+  profile = webdriver.FirefoxProfile()
+  profile.set_preference("webdriver.load.strategy", "unstable")
+  browser = webdriver.Firefox(firefox_profile=profile)
+  browser.get(embed)
+  time.sleep(3)
```

Comment:
Just out of curiosity, did you guess 3 seconds, or is it an established number from somewhere? Like, could it happen that someone's internet is slow af and 3 seconds isn't enough?